src/app/components/ModalCustomization/index.tsx

```diff
@@ -46,6 +48,7 @@ export const ModalCustomization = (props: Props) => {
     contentText,
     loading,
     textAgreeBtn,
+    userExamId,
```

Comment:
Don't edit this, this component is generic for many cases, don't define a specific value in this. You can use route-dom's navigation instead to redirect with click . event.

Reply:
but when click start button in popup duration, i used ModalCustomization component, if don't add the prop in here, how can i click button in popup to redirect page with link "exams/do-exam/${userExamId}"

Reply:
Same as above you can use navigation https://reactrouter.com/en/main/hooks/use-navigate to redirect.